test/WebJobs.Extensions.Tests/Extensions/Timers/Listener/TimerListenerTests.cs

```diff
@@ -355,6 +356,61 @@ public void Timer_VerifyMaxInterval()
             await RunInitialStatusTestAsync(null, "Function 'Program.TestTimerJob' initial status: Last='', Next='', LastUpdated=''");
         }
 
+        /// <summary>
+        /// Situation where the DST transition happens in the middle of the schedule, with the
+        /// next occurrence AFTER the DST transition.
+        /// </summary>
+        [Fact]
+        public void GetNextInterval_NextAfterDST_ReturnsExpectedValue()
+        {
+            // Running on the Friday before the DST switch at 2 AM on 3/11 (Pacific Standard Time)
+            // Note: this test uses Local time, so if you're running in a timezone where
+            // DST doesn't transition the test might not be valid.
+            var now = new DateTime(2018, 3, 9, 18, 0, 0, DateTimeKind.Local);
```

Comment:
Will this pass in AppVeyor? I believe it uses UTC.

Reply:
Yes they will pass there. Looks like my PR build passed?